WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -27,8 +27,16 @@ def mail_to_wca_board
     mail_to "board@worldcubeassociation.org", "Board", target: "_blank"
   end
 
-  def md(content)
-    Redcarpet::Markdown.new(Redcarpet::Render::HTML.new).render(content).html_safe
+  def md(content, target_blank: false)
+    options = {
+      hard_wrap: true
+    }
+
+    if target_blank
+      options[:link_attributes] = { rel: 'nofollow', target: "_blank" }
```

Comment:
I'm not sure that we want `rel: 'nofollow'` here. According to https://en.wikipedia.org/wiki/Nofollow, it seems to be something we'd use if we let anybody add links to our website, but we only let trusted people like delegates add links.